packages/compiler/src/server/types.ts

```diff
@@ -156,8 +168,8 @@ export interface SemanticToken {
 export type CustomRequestName =
   | "typespec/getInitProjectContext"
   | "typespec/initProject"
-  | "typespec/validateInitProjectTemplate";
-
+  | "typespec/validateInitProjectTemplate"
+  | "typespec/compileProject";
```

Comment:
```suggestion
  | "typespec/compile";
```

woudl just call that compile

Reply:
or `internal-compile` to aleviante concern about an api that isn't really meant for anybody else right now

Reply:
update to internalCompile